packages/@react-aria/list/docs/useList.mdx

```diff
@@ -0,0 +1,395 @@
+
+When the `selectionBehavior` prop is set to `"replace"`, clicking a row with the mouse _replaces_ the selection with only that row. Using the arrow keys moves both focus and selection. To select multiple rows, modifier keys such as <Keyboard>Ctrl</Keyboard>, <Keyboard>Cmd</Keyboard>, and <Keyboard>Shift</Keyboard> can be used. To move focus without moving selection, the <Keyboard>Ctrl</Keyboard> key on Windows or the <Keyboard>Option</Keyboard> key on macOS can be held while pressing the arrow keys. Holding this modifier while pressing the <Keyboard>Space</Keyboard> key toggles selection for the focused row, which allows multiple selection of non-contiguous items. On touch screen devices, selection always behaves as toggle since modifier keys may not be available. This behavior emulates native platforms such as macOS and Windows, and is often used when checkboxes in each row are not desired.
+
+```tsx example
+<PokemonList aria-label="List with replace selection behavior" selectionMode="multiple" selectionBehavior="replace" />
+```
+
+### Row actions
+
+`useList` supports row actions via the `onAction` prop, which is useful for functionality such as navigation. In the default `"toggle"` selection behavior, when nothing is selected, clicking or tapping the row triggers the row action.
+When at least one item is selected, the list is in selection mode, and clicking or tapping a row toggles the selection. Actions may also be triggered via the <Keyboard>Enter</Keyboard> key, and selection using the <Keyboard>Space</Keyboard> key.
+
+This behavior is slightly different in the `"replace"` selection behavior, where single clicking selects the row and actions are performed via double click. On touch devices, the action becomes the primary tap interaction,
+and a long press enters into selection mode, which temporarily swaps the selection behavior to `"toggle"` to perform selection (you may wish to display checkboxes when this happens). Deselecting all items exits selection mode
+and reverts the selection behavior back to `"replace"`. Keyboard behaviors are unaffected.
+
+```tsx example
+<PokemonList aria-label="List with replace selection behavior and row actions" selectionMode="multiple" selectionBehavior="replace" onAction={key => alert(`Opening item ${key}...`)} />
+```
+
+With `"toggle"` selection behavior, we can show checkboxes for selection while still having row actions. In this case, clicking the checkbox itself will toggle selection of that row, while single clicking anywhere else on the row will trigger the action.
+Once one row has been selected, clicking anywhere the rows will select or deselect them.
+
+```tsx example
+<PokemonList aria-label="List with toggle selection behavior and row actions" selectionMode="multiple" selectionBehavior="toggle" onAction={key => alert(`Opening item ${key}...`)} />
+```
+
```

Comment:
Maybe we should make the above examples and supporting text match the spectrum docs for ListView where it makes sense? Some of them are slightly different from table, and I think text around actions and select behavior has been updated.

Reply:
Updated the row actions example and text to better match ListView docs. I wasn't sure what would be a good alternative for selection, since ListView docs focus on `selectionStyle`, which is a spectrum prop. Take a look at that section when it builds and let me know if there are any specific changes that would be better

Reply:
IMO, you can borrow the following copy from the ListView docs from Selection Behavior:
```
In the highlight selection style, however, clicking a row with the mouse replaces the selection with only that row. Using the arrow keys moves both focus and selection. To select multiple rows, modifier keys such as Ctrl, Cmd, and Shift can be used. On touch screen devices, selection always behaves as toggle since modifier keys may not be available.

These selection styles implement the behaviors defined in [Aria Practices](https://w3c.github.io/aria-practices/#listbox_kbd_interaction).
```
At the very least, including the aria practices link will be helpful for the user in explaining all the interaction combinations